go.mod

```diff
@@ -1,79 +1,85 @@
 module github.com/harvester/harvester-network-controller
 
-go 1.18
+go 1.19
```

Comment:
it's better to make it consistent with ur https://github.com/harvester/network-controller-harvester/blob/master/Dockerfile.dapper#L1